### DISCUSSION TITLE
Windows build

### DISCUSSION THREAD

**Discussion by u01jsrc** [Ifsttar/I-Simpa#354] [General]
Hi,

I've been struggling building a windows exe and would welcome any advice from someone that's done it.

I've run through the install instructions, but not sure if they are up to date as getting errors when building in cmake. I've tried many many variations of the instructions and just can't get it building!

Lastly, as an alternative, I've tried to make a fork and do it in appveyor, via a Web browser. Getting issues there too.

Any and all tips welcome, as I'm keen to get the STI version as a Windows exe.

**Comment by u01jsrc** [comment]
I managed to get appveyor building, so very happy.

Next problem I'll try and figure out, is that a lot of it is in French where I'd like it in English. Not had a chance to play yet, but any tips welcome in the meantime.

**Reply by u01jsrc** [reply]
I forked this branch:

https://github.com/wbinek/I-Simpa/tree/Port_STI_calculation

to here, so that I could build my own exe (the original artifact exe's had been removed because of appveyor's retention policy):

https://github.com/u01jsrc/I-Simpa-STI

I am guessing it is to do with a lack of localisation files, but I am not sure, so any help welcome.

To be clear, I can set the language to English in i-Simpa (and reboot, of course), but a lot of it is still showing in French after that.

**Comment by nicolas-f** [comment]
Hi,

Its weird you have french because translation keys are in english:

https://www.transifex.com/LAE/i-simpa/viewstrings/#en/i-simpa-en/28867871

There is github actions now for windows and macos now so we may migrate to github actions for the windows/macos/linux release.

**Reply by u01jsrc** [reply]
Thanks

**Reply by u01jsrc** [reply]
Figured it out. I was loading the tutorials, which are all in French in this version, but when creating a New Project it's fine. So the program is in English, and I just need to copy over the English documentation, so problem solved.

Thanks again.
